src/js/letters/utils/helpers.jsx

```diff
@@ -15,6 +15,23 @@ export function apiRequest(resource, optionalSettings = {}, success, error) {
   return commonApiClient(requestUrl, optionalSettings, success, error);
 }
 
+export function invalidAddressProperty() {
+  return (
+    <div id="invalidAddress" className="usa-width-three-fourths">
+      <div className="usa-alert usa-alert-error">
+        <div className="usa-alert-body">
+          <h2 className="usa-alert-heading">Address unavailable</h2>
+          <p className="usa-alert-text">
+            We're encountering an error with your address information. This is not
+            required for your letters, but if you'd like to see the address we have
+            on file, or to update it, please visit <a href="/" target="_blank">this link</a>.
```

Comment:
Yep, we're actually going to be changing this content anyway so it's fine for now.